WatchParty/client/src/components/MoviePage/__test__/MoviePage.test.jsx

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import { mount } from 'enzyme';
+import { shallow } from 'enzyme';
```

Comment:
I wish we had giphy on github, this needs that please gif

Reply:
i have finally conquered unit testing